src/pretix/control/navigation.py

```diff
@@ -186,6 +186,14 @@ def get_event_navigation(request: HttpRequest):
                     }),
                     'active': 'event.items.questions' in url.url_name,
                 },
+                {
+                    'label': _('Discounts'),
+                    'url': reverse('control:event.items.discounts', kwargs={
+                        'event': request.event.slug,
+                        'organizer': request.event.organizer.slug,
+                    }),
+                    'active': 'event.items.discounts' in url.url_name,
+                },
```

Comment:
Before checking the source, it took me quite some time to find this menu item... I think we should consider moving this closer to the Vouchers menu item.

Reply:
Uh… interesting. But … they are not vouchers. Happy for further input here, eg from @wiffbi

Reply:
"closer" not "put it in there" ;-)

Even though those are two very different concepts, I feel like they somehow work together... :shrug:

Reply:
Hmm, I'm still not sure, especially since in another comment you ask for an option that prevents mixing of discounts and vouchers :see_no_evil: I feel that they are part of "pricing" and "pricing" is something that belongs as much to "products" as e.g. quotas do.

Also, what options do we have? If we don't want to put it under products and not under vouchers, it needs to be a top-level item? I don't feel that is warranted, more top-level items make it harder for new users to get an overview and this is *not* a feature you need to discover before you disovered quotas.

Reply:
Hm, if you say it like that, I do agree 100% - it is part of the pricing and that does indeed live within the products-area...

Sometimes it helps just talking about it ;-)